tgg/seqr_scripts/pull_seqr_pedigrees.py

```diff
@@ -29,19 +29,24 @@ def pull_project_peds(session_id: str, projects: set):
             )
             for project_guid in projects:
                 project_guid = project_guid.rstrip()
-                r = s.get(
-                    f"{SEQR_URL}/api/project/{project_guid}/details"
+                family_r = s.get(
+                    f"{SEQR_URL}/api/project/{project_guid}/get_families"
                 )
-
-                if r.status_code != requests.codes["ok"]:
+                individual_r = s.get(
+                    f"{SEQR_URL}/api/project/{project_guid}/get_individuals"
+                )
+                if family_r.status_code != requests.codes["ok"]:
+                    logger.info(f"Could not find {project_guid} in seqr")
```

Comment:
should the error messages here and on line 42 be different? maybe something like:
```
Could not find {project_guid} when searching for family IDs
```
(and individual IDs below)